ceph.spec.in

```diff
@@ -556,6 +557,10 @@ Requires:	libradosstriper1 = %{_epoch_prefix}%{version}-%{release}
 %if 0%{?suse_version}
 Requires(pre):	pwdutils
 %endif
+%if 0%{with ubbd}
+Requires:	ubbd
+BuildRequires:	ubbd-devel
```

Comment:
Do we need this `BuildRequires` if we always download ubdb sources when building?

Reply:
Yes, good catch, that's for last version of this patch. thanx @trociny